### PR TITLE
Deprecate outdated utils/synchronized reference

### PR DESCRIPTION
**Pull request by kiip1**

### CODE REVIEW

**Review by Moderocky — COMMENTED** [review]
This looks correct but I feel like it would be good for somebody to test it works first.

File: src/main/java/ch/njol/util/Predicate.java [hunk 1]
```diff
@@ -18,13 +18,18 @@
  */
 package ch.njol.util;
 
+import org.jetbrains.annotations.ApiStatus;
+
 import javax.annotation.Nullable;
 
 /**
  * @author Peter G�ttinger
  *
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/util/Predicate.java:27]:
```suggestion
 * @deprecated Use {@link java.util.function.Predicate}
```

File: src/main/java/ch/njol/util/Setter.java [hunk 2]
```diff
@@ -18,11 +18,19 @@
...
- */
-public interface Setter<T> {
-	
-	public void set(T t);
-	
+import org.jetbrains.annotations.ApiStatus;
+
+import java.util.function.Consumer;
+
+@Deprecated
```
Comment (APickledWalrus) [inline src/main/java/ch/njol/util/Setter.java:25]:
```suggestion
/**
 * Use {@link Consumer}
 */
@Deprecated
```

**Review by APickledWalrus — CHANGES_REQUESTED** [review]
If this happens to get taken over, I just have a function documentation-related thoughts. I am not a huge fan of the VectorMath changes, as there is a lot of method duplication going on now.

**Comment by APickledWalrus** [comment]
Given this is rather outdated, and seems to mostly be duplicated/featured in https://github.com/SkriptLang/Skript/pull/6670, I will be closing it.

**Pull request closed without merging**